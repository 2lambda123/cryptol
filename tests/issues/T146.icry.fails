Known problem. See #146.
See also #323, which seems to be another instance of the same problem.
