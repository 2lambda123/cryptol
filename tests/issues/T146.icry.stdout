Loading module Cryptol
Loading module Cryptol
Loading module Main

[error] at ./T146.cry:1:18--6:10:
  The type ?r33 is not sufficiently polymorphic.
    It cannot depend on quantified variables: a`862
  where
  ?r33 is type of field 'v1'
          at ./T146.cry:3:9--6:10
[error] at ./T146.cry:5:19--5:24:
  The type ?s33 is not sufficiently polymorphic.
    It cannot depend on quantified variables: a`862
  where
  ?s33 is type of field 'v2'
          at ./T146.cry:3:9--6:10
